app/build.yaml

```diff
@@ -0,0 +1,19 @@
+# Read about `build.yaml` at https://pub.dartlang.org/packages/build_config
+# To update generated code, run `pub run build_runner build`
+targets:
+  pub_dartlang_org:
+    sources:
+      - 'lib/shared/*.dart'
+      - 'lib/search/backend*.dart'
+    builders:
+      json_serializable:
```

Comment:
So, do `json_serializable` do both `JsonSerializableGenerator` and `JsonLiteralGenerator`?

Reply:
I forget – but we don't need the LiteralGenerator any more